src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionAccident.java

```diff
@@ -0,0 +1,55 @@
+package org.umssdiplo.automationv01.stepdefinitionproject;
+
+import cucumber.api.DataTable;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.testng.Assert;
+import org.umssdiplo.automationv01.core.managepage.Accident.Accident;
+import org.umssdiplo.automationv01.core.managepage.Accident.CreateAccident;
+import org.umssdiplo.automationv01.core.managepage.Login.Login;
+import org.umssdiplo.automationv01.core.utils.LoadPage;
+
+import java.util.List;
+
+public class StepsDefinitionAccident {
+    private Accident accident = new Accident();
+    private int accidents;
+    private CreateAccident createAccident = new CreateAccident();
+
+    @When("^go to accident Page$")
+    public void goToAccident() throws Throwable {
+        accident.selectAccidentLink();
+    }
+
+    @And("^get list of accidents$")
+    public void getAccidents() throws Throwable {
+        accidents = accident.countAccident();
+    }
+
+    @And("^go to create accident page$")
+    public void createAccident() throws Throwable {
+        accident.createAccident();
+    }
+
+    @And("^fill all required fields$")
+    public void enterDescription$(DataTable accidentInformation) throws Throwable {
+        List<List<String>> data = accidentInformation.raw();
```

Comment:
Use an object that will give you that structure that you need

Reply:
int scenario1AmountAccident;  // variable class

scenario1AmountAccident = data.size();